tls: add code for ERR_TLS_INVALID_PROTOCOL_METHOD (#24729)
nodejs/node: doc/api/errors.md

A TLS/SSL handshake timed out. In this case, the server must also abort the
connection.

<a id="ERR_TLS_INVALID_PROTOCOL_METHOD"></a>
### ERR_TLS_INVALID_PROTOCOL_METHOD

> An invalid `secureProtocol` method was specified. It was either unknown, or

Reviewer: ```suggestion
The specified  `secureProtocol` method is invalid. It is  either unknown, or
```

Reviewer: ☝️ Just a suggestion. Ignore it if you don't like it. I think it's simpler and more direct, but whatevs.
